editor.planx.uk/src/pages/Teams.tsx

```diff
@@ -1,71 +1,102 @@
-import Edit from "@mui/icons-material/Edit";
-import Visibility from "@mui/icons-material/Visibility";
 import Box from "@mui/material/Box";
 import Card from "@mui/material/Card";
+import Container from "@mui/material/Container";
 import { styled } from "@mui/material/styles";
 import Typography from "@mui/material/Typography";
 import { Team } from "@opensystemslab/planx-core/types";
 import React from "react";
 import { Link } from "react-navi";
+import Dashboard from "ui/editor/Dashboard";
 
 import { useStore } from "./FlowEditor/lib/store";
 
+interface TeamTheme {
+  slug: string;
+  primaryColour: string;
+}
+
 interface Props {
   teams: Array<Team>;
+  teamTheme: Array<TeamTheme>;
 }
 
-const Root = styled(Box)(({ theme }) => ({
-  backgroundColor: theme.palette.background.dark,
-  color: "#fff",
+export const Root = styled(Box)(({ theme }) => ({
+  backgroundColor: theme.palette.background.default,
   width: "100%",
-  flex: 1,
+  display: "flex",
+  alignItems: "flex-start",
+  flexGrow: 1,
+}));
+
+const StyledLink = styled(Link)(() => ({
+  textDecoration: "none",
+}));
+
+const TeamCard = styled(Card)(({ theme }) => ({
+  display: "flex",
   justifyContent: "flex-start",
   alignItems: "center",
+  marginBottom: theme.spacing(2),
+  color: theme.palette.text.primary,
+  outline: `1px solid ${theme.palette.border.light}`,
+  outlineOffset: "-1px",
+  borderRadius: "1px",
 }));
 
-const Dashboard = styled(Box)(({ theme }) => ({
-  backgroundColor: theme.palette.background.dark,
-  color: "#fff",
-  width: "100%",
-  maxWidth: 600,
-  margin: "auto",
-  padding: theme.spacing(8, 0, 4, 0),
+const TeamColourBand = styled(Box)(({ theme }) => ({
+  display: "flex",
+  alignSelf: "stretch",
+  width: theme.spacing(1.5),
+  zIndex: 1,
 }));
 
-const StyledLink = styled(Link)(() => ({
-  textDecoration: "none",
-}));
+const Teams: React.FC<Props> = ({ teams, teamTheme }) => {
+  const canUserEditTeam = useStore.getState().canUserEditTeam;
 
-const Teams: React.FC<Props> = ({ teams }) => {
+  const editableTeams = teams.filter((team) => canUserEditTeam(team.slug));
+  const viewOnlyTeams = teams.filter((team) => !canUserEditTeam(team.slug));
```

Comment:
nit: You could also do this in a single iteration with a `.reduce()` or `.forEach()`

**reduce()**
```ts
const { editableTeams, viewOnlyTeams } = teams.reduce(
  (acc, team) => {
    canUserEditTeam(team.slug)
      ? acc.editableTeams.push(team)
      : acc.viewOnlyTeams.push(team)
    
    return acc;
  },
  { editableTeams: [], viewOnlyTeams: [] }
);
```

**forEach()**
```ts
const editableTeams = [];
const viewOnlyTeams = [];

teams.forEach((team) => (
  canUserEditTeam(team.slug)
    ? editableTeams.push(team)
    : viewOnlyTeams.push(team)
));
```

Reply:
Thanks for this, I've gone with `forEach` as it reads better as a piece of logic.